src/modules/mc_att_control/mc_att_control_main.cpp

```diff
@@ -518,15 +518,15 @@ MulticopterAttitudeControl::control_attitude_rates(float dt)
 	rates(1) -= _sensor_bias.gyro_y_bias;
 	rates(2) -= _sensor_bias.gyro_z_bias;
 
-	math::Vector<3> rates_p_scaled = _rate_p.emult(pid_attenuations(_tpa_breakpoint_p.get(), _tpa_rate_p.get()));
-	//math::Vector<3> rates_i_scaled = _rate_i.emult(pid_attenuations(_tpa_breakpoint_i.get(), _tpa_rate_i.get()));
-	math::Vector<3> rates_d_scaled = _rate_d.emult(pid_attenuations(_tpa_breakpoint_d.get(), _tpa_rate_d.get()));
+	Vector3f rates_p_scaled = _rate_p.emult(pid_attenuations(_tpa_breakpoint_p.get(), _tpa_rate_p.get()));
+	//Vector3f rates_i_scaled = _rate_i.emult(pid_attenuations(_tpa_breakpoint_i.get(), _tpa_rate_i.get()));
```

Comment:
@MaEtUgR we don't have to resolve it in this PR, but did you notice this line? We shouldn't carry around commented out code, so it would be good to address at some point.

Reply:
@dagar I just saw that it was ignored by @bkueng and you before and assumed everyone knows why it's commented out. After all you commented it: https://github.com/PX4/Firmware/commit/d6e9287f516aee8fa69ca571a400090de15e58ba 😄 

Now I found the error: #9166